sdk/core/core-amqp/src/cbs.ts

```diff
@@ -207,9 +207,9 @@ export class CbsClient {
       return this._fromAmqpMessageResponse(responseMessage);
     } catch (err) {
       logger.warning(
-        "[%s] An error occurred while negotiating the cbs claim: %O",
+        "[%s] An error occurred while negotiating the cbs claim: %s",
```

Comment:
What was wrong with "errdr", looks perfectly fine to me! :)

Reply:
I'm not sure I understand the question...

Reply:
nevermind, the joke is lost